tools/cockpit.spec

```diff
@@ -149,7 +149,7 @@ The Cockpit Web Service listens on the network, and authenticates users.
 env NOCONFIGURE=1 ./autogen.sh
 %endif
 %configure --disable-static --disable-silent-rules --with-cockpit-user=cockpit-ws --with-branding=%{branding}
-make -j %{?extra_flags} all
+make %{?extra_flags} all
```

Comment:
Can we specify -j8 or something like that? Not having parallel builds really kills build performance.